### ISSUE TITLE
USB3 port?

### ISSUE THREAD

**Issue by servetcoskun** [georgealexakis/multispectral_processing#1]
Is it possible to port the solution to the same camera but with the USB3 connection?

**Comment by georgealexakis** [comment]
Yes , I think it is possible, but i have not tested it yet. This depends on the driver package of the camera. Check the link of the package: http://wiki.ros.org/ueye_cam.
